File: rest.c
// C Example:

# include <stdio.h>

int main(){

    // defining The Dividend, Divisor, Rest and Quotient
    int D, d, R, Q;
    printf("Say two numbers, that the first will be divided by the second for discover the rest of solution.\n");

    scanf("%d %d",&D,&d);
    printf("\n");

    Q = D/d;
    R = D - Q*d;

    printf("The Rest of %d/%d is %d, and the Result is %d.\n",D,d,R,Q);
}

File: rest.py
# Python example:
class Return:
	def play():
		print("Say two numbers, that the first will be divided by the second for discover the rest of solution.")
		
		D = int(input("\n» "))
		d = int(input("\n» "))
		return(D, d)

	def ret(Divisor, divided):
		Q, R = 0, 0
		for i in range(D):
			i = i + 1
			r = i*d
			if r > D:
				Q = i - 1
				R = D - (d*Q)
				return Q, R
				break
			else:
				continue
				
print(Return.ret(Return.play()))
